schema.graphql

```diff
@@ -1,28 +1,29 @@
 "https://thegraph.com/docs/en/developer/create-subgraph-hosted/"
-type Account @entity {
-    "user information"
+type User @entity {
+    "user address"
     id: ID!
-    referenceCode: String!
-    isVIP: Boolean!
-    auths: [Auth!] @derivedFrom(field: "account")
+    referenceCode: String
+    isVIP: Boolean
+    authedTokens: [AuthedToken!]
     whitelisted: [RoundWinner!] @derivedFrom(field: "winner")
+    userJoinedRoundList: [UserRoundTicket!]
 }
 
-type Auth @entity {
-    "user authorized tokens"
+type AuthedToken @entity {
+    "tokenAddr + tokenID"
     id: ID!
-    account: Account!
+    user: User!
     txHash: String!
     tokenAddr: String!
     tokenID: Int!
 }
 
 type Bucket @entity {
```

Comment:
Maybe we need start epoch & end epoch for bucket.

Reply:
I thought we discussed that, we will have the second day after creation as bucket start epoch, and since there is only one round, end epoch will be the next day?

since we already have the creation time, maybe we can derive from that.

if in the future we want to allow customized start date, we will add these fields. what do you think?